WordprocessingML/Wml2Ew/Program.cs

```diff
@@ -102,6 +103,14 @@ private static bool CheckInputParameters(string[] args, out string xmlFileName,
                 return false;
             }
 
+            bool isValid = Guid.TryParseExact(itemGuid, "D", out _);
+
+            if (itemGuid != null && isValid != true)
```

Comment:
Tady by bylo lepší to volat takto, ať zbytečně nevoláš TryParse s null hodnotou:

`
if (itemGuid != null && !Guid.TryParseExact(itemGuid, "D", out _))
`